### PR TITLE
Include Swagger Documentation to the proyect

### PR DESCRIPTION
**Pull request by jorge97fernandez**
Add both JSON and HTML Swagger Documentation to the proyect

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -166,6 +166,11 @@ The syntax of the @Value annotation is
 If the argument is "/", it means that the method whose annotation is @GetMapping("/") will be triggered when a request is made to the "root" of the web.
 If the argument is of the type "string/string...etc", it means that the method whose annotation is @GetMapping("string/string...") will be triggered when a request is made to this endpoint of the web.
 
+### @Configuration
+The @ Configuration annotation is used to indicate that the class that follows it, has one or more @Bean methods. The @Bean annotation defines objects that are
```
Comment (fjlopez) [inline description.md]: Headers must be surrounded by blank lines.

File: description.md [hunk 2]
```diff
@@ -166,6 +166,11 @@ The syntax of the @Value annotation is
 If the argument is "/", it means that the method whose annotation is @GetMapping("/") will be triggered when a request is made to the "root" of the web.
 If the argument is of the type "string/string...etc", it means that the method whose annotation is @GetMapping("string/string...") will be triggered when a request is made to this endpoint of the web.
 
+### @Configuration
+The @ Configuration annotation is used to indicate that the class that follows it, has one or more @Bean methods. The @Bean annotation defines objects that are 
+managed by the Spring Container.
+
+
```
Comment (fjlopez) [inline description.md]: Remove the extra blank line here.

File: src/main/java/es/unizar/webeng/hello/Application.java [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-    public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
-    }
-
-    @Override
-    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
-        return application.sources(Application.class);
-    }
-}
+package es.unizar.webeng.hello;
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/Application.java]: You must avoid to replace an identical file in a pull request.

File: description.md (hunk 2 above)
Comment (jorge97fernandez) [reply description.md]: Documentation fixed

File: src/main/java/es/unizar/webeng/hello/Application.java (hunk 3 above)
Comment (jorge97fernandez) [reply src/main/java/es/unizar/webeng/hello/Application.java]: Issue fixed

**Pull request merged**